Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Driver.hs

```diff
@@ -398,6 +400,8 @@ collectCash merchantShortId reqDriverId = do
   Esq.runTransaction $ do
     QDF.updateStatus COLLECTED_CASH driverFee.id now
     QDFS.clearPaymentStatus (cast driverFee.driverId) driverInfo_.active
+  let totalDriverFee = fromIntegral driverFee.govtCharges + fromIntegral driverFee.platformFee.fee + driverFee.platformFee.cgst + driverFee.platformFee.sgst
```

Comment:
Why is this required ?

Reply:
need to calculate final fee to send in SMS. The DriveFee table does not contain actual amount paid